source/extensions/filters/network/dubbo_proxy/deserializer.h

```diff
@@ -1,23 +1,52 @@
 #pragma once
 
+#include <string>
+#include <unordered_map>
+
 #include "envoy/buffer/buffer.h"
 
+#include "common/common/assert.h"
+#include "common/config/utility.h"
 #include "common/singleton/const_singleton.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace NetworkFilters {
 namespace DubboProxy {
 
+enum class DeserializerType : uint8_t {
+  Hessian = 0,
+  Json = 1,
+
+  // ATTENTION: MAKE SURE THIS REMAINS EQUAL TO THE LAST PROTOCOL TYPE
+  LastDeserializerType = Json,
```

Comment:
Didn't see where it was used, can it be removed here?

Reply:
ok, i deleted it